DataDonationCryptoObjC/DataDonationCryptoObjC/Sources/DataDonationCryptoObjC.swift

```diff
@@ -16,9 +16,22 @@
 
 import Foundation
 
-@objc public protocol KeychainKeyProviderProtocol {
-    @objc func getDonorPrivateKey(for programName: String) throws -> String
-    @objc func getDonorPublicKey(for programName: String) throws -> String
-    @objc func removeDonorKeyPair(for programName: String) throws
-    @objc func storeDonorKeyPairData(_ data: Data, for programName: String) throws
+@objc public protocol DataDonationCryptorObjCProtocol {
+    @objc func encrypt(_ plainBody: Data, base64EncodedPublicKey: String) throws -> Data 
+    @objc func decrypt(_ encryptedData: Data, base64EncodedPrivateKey: String) throws -> Data
 }
+
+@objc public protocol DataDonationSignerObjCProtocol {
+    @objc func sign(data: Data, isSalted: Bool, donorKeyIdentifier: String) throws -> Data
+    @objc func verify(data: Data, signature: Data, isSalted: Bool, donorKeyIdentifier: String) throws
+}
+
+@objc public protocol DataDonationKeychainStoreObjCProtocol {
+    @objc func generateDonorKeyPair(with keyIdentifier: String) throws
+    @objc func storeDonorKeyPairData(_ data: Data, with keyIdentifier: String) throws
+    @objc func fetchDonorPrivateKeyAsBase64(with keyIdentifier: String) throws -> String
+    @objc func fetchDonorPublicKeyAsBase64(with keyIdentifier: String) throws -> String
```

Comment:
I am a bit uncertain about this - can we bundle it with generate and return simply a pair of 2 Strings? Since we have to call generate always with both fetch methods.